iml-wasm-components/iml-action-dropdown/src/lib.rs

```diff
@@ -73,7 +113,8 @@ fn view(state: seed::App<Msg, Model>, model: &Model) -> El<Msg> {
     };
 
     let next_open = Msg::Open(!model.open);
-    let lock_count = locks::lock_list(&model.locks, &model.records).len();
+    let records = model.records.clone();
```

Comment:
Why clone here?

Reply:
My first attempt was to do the following:
```
did_mount(move |_| spawn_local(get_actions(state.clone(), &model.records.clone())))
```
However, I get the following error message:
```
Explicit lifetime required in the type of 'model'
```
I did play around with an explicit lifetime but I wasn't having much luck.